sQuick.package/MainInterface.class/class/run.st

```diff
@@ -1,8 +1,3 @@
 as yet unclassified
 run
-	(instance == nil) ifTrue: [
-		instance := MainInterface  new.
-		^ instance.
-	]; ifFalse: [
-		^ instance.
-	]
\ No newline at end of file
+	^ instance ifNil: [ instance := self new ]
```

Comment:
This is Pharo's canonical lazy initialization pattern. Memorize it - you will see it a lot!